src/OneExplorer/OneExplorer.ts

```diff
@@ -559,6 +559,18 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
       vscode.commands.registerCommand("one.explorer.runCfg", (node: Node) => {
         vscode.commands.executeCommand("one.toolchain.runCfg", node.uri.fsPath);
       }),
+      vscode.commands.registerCommand(
+        "one.explorer.runSingleSelectedCfg",
+        () => {
+          let selectedCfg = provider.getSelectedCfg();
+          if (selectedCfg && selectedCfg.length === 1) {
```

Comment:
(sorry for adding comments one-by-one)
I'm not sure about current UI guide but I think running this with no selection or with two or more selected items won't do any action, resulting user to just wait and wait...
May be add a `TODO: handle for none or multiple selection` for now?

Reply:
ok

Reply:
fixed